api_tests/search/serializers/test_serializers.py

```diff
@@ -0,0 +1,64 @@
+from modularodm import Q
+from nose.tools import *  # flake8: noqa
+
+from api.search.serializers import SearchSerializer
+from api_tests.utils import create_test_file
+
+from tests.base import DbTestCase
+from tests.factories import (
+    AuthUserFactory,
+    NodeFactory,
+    ProjectFactory,
+)
+from tests.utils import make_drf_request, mock_archive
+
+from website.models import MetaSchema
+from website.project.model import ensure_schemas
+from website.project.metadata.schemas import LATEST_SCHEMA_VERSION
+from website.search import search
+
+
+class TestSearchSerializer(DbTestCase):
+
+    def setUp(self):
+        super(TestSearchSerializer, self).setUp()
+        search.delete_all()
```

Comment:
This seems like something that should happen in the tearDown(). Is it clearing out things from the super() call?